### PR TITLE
APP-3750: Fix Register purchase to mock autorenew so we can have a pu…

### PR DESCRIPTION
**Pull request by laurenyew-nytimes**
…rchase return SKU type (autorenew = true --> subscription)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Change Log
 ==========
 
+Version 0.0.8
+-------------
+- [APP-3750](https://jira.nyt.net/browse/APP-3750) Update Purchase to include `autorenew` mock so can have sku type in Purchase
+
```
Comment (wdziemia) [inline CHANGELOG.md:7]: This version is already on sonatype

https://mvnrepository.com/artifact/com.nytimes.android/register

File: lib/src/main/java/com/nytimes/android/external/registerlib/InAppPurchaseData.kt [hunk 2]
```diff
@@ -49,6 +50,10 @@ class InAppPurchaseData {
         return purchaseToken
     }
 
+    fun isAutoRenewing(): Boolean? {
+        return isAutoRenewing
+    }
```
Comment (wdziemia) [inline lib/src/main/java/com/nytimes/android/external/registerlib/InAppPurchaseData.kt:55]: Can you add a test for this function?

Comment (laurenyew-nytimes) [reply lib/src/main/java/com/nytimes/android/external/registerlib/InAppPurchaseData.kt:55]: Done!

File: CHANGELOG.md (hunk 1 above)
Comment (laurenyew-nytimes) [reply CHANGELOG.md]: Bumped version to 0.0.9

**Review by wdziemia — APPROVED** [review]

**Pull request merged**